### ISSUE TITLE
New element to Patron entity to specify money owing

### ISSUE THREAD

**Issue by kjeyakanthan** [bic-org-uk/bic-lcf#142]
On return on a patron request, would it be possible to supply back the actual money owing for this patron. 

This will allow systems that do payments, to do a simple "Pay All" function, without first having to retrieve the associated charges, which may be quite an expensive operation.

**Comment by franciscave** [comment]
At present, to determine the actual money owing by a Patron, there would need to be a request for details of all charges not yet paid and the calculation would have to be done locally at the terminal. 

In LCF v1.0.1 we added E03C31 Deposit balance to the Patron entity, but we didn't add a similar structure containing a charge balance, although I seem to remember that we discussed this. I believe we decided against adding a charge balance, because this would involve duplication of data and the possibility of a recording error leading to an inconsistency between a charge balance on the Patron record and the sum or charges not yet paid and owing by that Patron. Do others agree?
